modules/installation-special-config-storage.adoc

```diff
@@ -46,72 +79,82 @@ Mirroring is available only for user-provisioned infrastructure deployments on {
-== Configuring disk encryption or mirroring
+== Configuring disk encryption and mirroring
 
-.Procedure
+You can enable and configure encryption and mirroring during an {product-title} installation.
 
-To enable boot disk encryption or mirroring during {product-title} deployment:
+.Prerequisites
+
+* You downloaded the {product-title} installation program on your installation node.
+* You installed Butane on your installation node.
++
+[NOTE]
+====
+Machine configs are used to configure the control plane and compute machines. In some situations, such as boot disk mirroring, you can use Butane to create the machine configs for you, making node configuration easier. See the _Creating machine configs with Butane_ section for more information.
+====
++
+* You have access to a {op-system-base-full} 8 system that can be used to generate a thumbprint of the Tang exchange key.
+
+.Procedure
 
 . If you want to use TPM v2 to encrypt your cluster, check to see if TPM v2 encryption needs to be enabled in the BIOS on each node. This is required on most Dell systems. Check the manual for your computer.
 
 . If you want to use Tang to encrypt your cluster, follow these preparatory steps:
 
-.. Access a Red Hat Enterprise Linux server from which you can configure the encryption settings.
+.. Set up a Tang server or access an existing one. See link:https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html/security_hardening/configuring-automated-unlocking-of-encrypted-volumes-using-policy-based-decryption_security-hardening#network-bound-disk-encryption_configuring-automated-unlocking-of-encrypted-volumes-using-policy-based-decryption[Network-bound disk encryption] for instructions. See link:https://youtu.be/2uLKvB8Z5D0[Securing Automated Decryption New Cryptography and Techniques] for a presentation on Tang.
 
-.. Set up or access an existing Tang server. See link:https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html/security_hardening/configuring-automated-unlocking-of-encrypted-volumes-using-policy-based-decryption_security-hardening#network-bound-disk-encryption_configuring-automated-unlocking-of-encrypted-volumes-using-policy-based-decryption[Network-bound disk encryption] for instructions. See link:https://youtu.be/2uLKvB8Z5D0[Securing Automated Decryption New Cryptography and Techniques] for a presentation on Tang.
-
-.. Install the `clevis` package, if it is not already installed:
+.. Install the `clevis` package on a {op-system-base} 8 server, if it is not already installed:
 +
 [source,terminal]
 ----
-$ sudo yum install clevis -y
+$ sudo yum install clevis
 ----
 
-.. Generate a thumbprint for the Tang server.
-
-... In the following command, replace the value of `url` with the Tang server URL:
+.. On the {op-system-base} 8 server, run the following command to generate a thumbprint of the exchange key. Replace `http://tangserver.example.com:7500` with the URL of your Tang server:
```

Comment:
Do we have a way to format the tangserver URL so that it is not a live link?

Reply:
Thank you for highlighting this. Following a discussion within the team, I have now added the `\` escape character before the URL, which prevents it from becoming a live link.